### PR TITLE
databases migrations

### PR DESCRIPTION
**Pull request by at-huydoan**
edit tables users 
add tables : categories, products, orders, orders_details

### CODE REVIEW

File: database/migrations/2014_10_12_000000_create_users_table.php [hunk 1]
```diff
@@ -14,9 +14,12 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->increments('id');
-            $table->string('name');
+            $table->string('username');
             $table->string('email')->unique();
             $table->string('password', 60);
+            $table->bigInteger('is_admin')->default(0);
```
Comment (ghost) [inline database/migrations/2014_10_12_000000_create_users_table.php]: it's better if you separate `users` and `admin_users` tables.

File: database/migrations/2014_10_12_000000_create_users_table.php [hunk 2]
```diff
@@ -14,9 +14,12 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->increments('id');
-            $table->string('name');
+            $table->string('username');
```
Comment (ghost) [inline database/migrations/2014_10_12_000000_create_users_table.php]: `username` should be `unique` also

File: database/migrations/2016_03_22_085459_create_categories_table.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('categories', function (Blueprint $table) {
```
Comment (ghost) [inline database/migrations/2016_03_22_085459_create_categories_table.php]: You must `drop` table in `down` function

File: database/migrations/2016_03_22_090010_create_products_table.php [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    {
+        Schema::create('products', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name');
+            $table->string('keywords');
+            $table->integer('quantity')->unsigned();
+            $table->decimal('price');
+            $table->string('description');
+            $table->string('image');
+            $table->integer('categories_id')->unsigned();
```
Comment (ghost) [inline database/migrations/2016_03_22_090010_create_products_table.php]: foreign key should be named in single not plural, `category_id`, not `categories_id`

File: database/migrations/2016_03_22_090824_create_orders_table.php [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('orders', function (Blueprint $table) {
+            $table->increments('id');
+            $table->integer('users_id')->unsigned();
+            $table->bigInteger('status');
```
Comment (ghost) [inline database/migrations/2016_03_22_090824_create_orders_table.php]: why `status` is `bigInteger` ?

File: database/migrations/2016_03_22_090824_create_orders_table.php [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+            $table->dateTime('date');
```
Comment (ghost) [inline database/migrations/2016_03_22_090824_create_orders_table.php]: what is the meaning of `date` column ?

File: database/migrations/2016_03_22_091319_create_orders_details_table.php [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('orders_details', function (Blueprint $table) {
+            $table->increments('id');
+            $table->integer('orders_id')->unsigned();
+            $table->integer('products_id')->unsigned();
+            $table->integer('qty_pro_ordered')->unsigned();
```
Comment (ghost) [inline database/migrations/2016_03_22_091319_create_orders_details_table.php]: why you named `qty_pro_ordered`, `price_pro_ordered` it's difficult to understand

**Comment by ghost** [comment]
@thienvuong please check my comment

**Comment by at-huydoan** [comment]
@at-vienlm please review this PR

**Pull request merged**